src/platform_impl/linux/wayland/window/state.rs

```diff
@@ -840,6 +847,28 @@ impl WindowState {
         }
     }
 
+    /// Make window background blurred
+    #[inline]
+    pub fn set_blur(&mut self, blurred: bool) {
+        if blurred && self.blur.is_none() {
+            self.blur = self
+                .blur_manager
+                .as_ref()
+                .map(|kbm| kbm.blur(self.window.wl_surface(), &self.queue_handle));
+            if let Some(blur) = &self.blur {
+                blur.commit();
```

Comment:
Not setting the region makes it to blue the entire window, right?

Reply:
Does that for me, also should be like this per docs and other usage examples